action.go

```diff
@@ -368,6 +368,14 @@ func (a Action) VerifyACL(acl ACL, keyring io.Reader) (err error) {
 		fingerprints = append(fingerprints, fp)
 	}
 
+	if len(fingerprints) == 0 {
+		return errors.New("No valid fingerprints found.")
+	}
+
+	if !verifyACLs {
```

Comment:
If we invert the logic above (require true to bypass ACL check) we'd need to also switch this around.